src/rcw/rcw.service.ts

```diff
@@ -715,6 +722,7 @@ export class RcwService {
     // delete the pdf file to freee up storage
     try {
       await fs.promises.unlink(filePath);
+      await fs.promises.unlink(outputPath);
```

Comment:
since we are deleting multiple files now, there should be some faster way to combine this statement and one above it.

Reply:
Updated to use Promises.all